webui/react/src/components/TrialLogFilters.tsx

```diff
@@ -0,0 +1,153 @@
+import { Button } from 'antd';
```

Comment:
**suggestion:** I would place this under `/src/pages/TrialLogs` (create a directory). Components under `/src/pages/X` indicate that the component is uniquely for that `X` page only and not truly shared.